docs-chef-io/content/inspec/cli.md

```diff
@@ -1,17 +1,10 @@
 +++
-title = "InSpec CLI"
-draft = false
-gh_repo = "inspec"
-
-[menu]
-  [menu.inspec]
-    title = "InSpec Executable"
-    identifier = "inspec/reference/cli.md InSpec Executable"
-    parent = "inspec/reference"
-    weight = 10
+title: "InSpec CLI"
+draft: false
+gh_repo: "inspec"
```

Comment:
Metadata should have equals signs to match the TOML spec, e.g.:

title = "InSpec CLI"